it/src/main/scala/com/wavesplatform/it/api/AsyncHttpApi.scala

```diff
@@ -93,6 +93,9 @@ object AsyncHttpApi extends Assertions {
     def getOrderStatus(asset: String, orderId: String): Future[MatcherStatusResponse] =
       matcherGet(s"/matcher/orderbook/$asset/WAVES/$orderId", waitForStatus = true).as[MatcherStatusResponse]
 
+    def getOrderStatus1(asset: String, orderId: String): Future[MatcherStatusResponse] =
```

Comment:
Instead of declaring a new method, please just add a default argument to the original one.